frontend/src/models/feedback.ts

```diff
@@ -0,0 +1,47 @@
+import { createModel } from '@rematch/core'
+import { graphQLGetErrors, graphQLCatchError } from '../services/graphQL'
+import { ApplicationState } from '../store'
+import { RootModel } from './rootModel'
+
+type FeedbackParams = { [key: string]: any }
+
+type IFeedbackState = {
+  sending: boolean
+  sent: boolean
+}
+
+const state: IFeedbackState = {
+  sending: false,
+  sent: false,
+}
+
+export default createModel<RootModel>()({
+  state,
+  effects: dispatch => ({
+    async sendFeedback() {
+      const { set } = dispatch.feedback
+
+      set({ sending: true })
+      try {
+        // TO DO
+        // const response = await graphQLSendFeedback({ })
+        // const errors = graphQLGetErrors(response)
+        // if (!errors) {
```

Comment:
Form should submit to ZenDesk API. Please see ticket description.
https://developer.zendesk.com/api-reference/
Thank you!